src/assets/styles/pages/homepage.scss

```diff
@@ -0,0 +1,169 @@
+body {
+ width: 100%;
+ height: 100vh;
+ overflow: hidden;
+ box-sizing: border-box;
+ font-family: 'Poppins', sans-serif;
+}
+
+.header {
+  position: fixed;
+  inset: 0 0 auto 0;
+  font-weight: 600; 
+  display: flex;
+  z-index: 1000;
+  
+
+
+  .home-nav {
+   display: flex;
+   align-self: center;
+   -webkit-box-align: stretch;
+   align-items: stretch;
+   height: 60px;
+   margin: 0 auto;
+   width: 100%;
+   max-width: 1320px;
+
+   .logo-link {
+    display: flex;
+    text-decoration: none;
+    align-items: center;
+    padding: 1rem;
+    cursor: pointer;
+
+   }
+
+   .btns {
+    display: flex;
+
+    .home-btn {
+     display:flex;
+     align-items: stretch;
+     align-self: stretch;
+     padding: 1.25rem 1rem 1rem;
+     font-size: 1rem;
+     border: none;
+     background-color: transparent;
+     line-height: 24px;
+     text-size-adjust: 100%;
+     font-stretch: 100%;
+     color: #172b4d;
+     cursor: pointer;
+    
+    &:hover {
+      color: #0065ff;
+    }
+    
+    }
+   }
+
+   .nav-link {
+    display: flex;
+    margin: 0 0 0 auto;
+    align-items: center;
+    padding: 0.5rem 5rem;
+    background-color:var(--dynamic-button-highlighted-text);
+    color:var(--dynamic-text);
+    text-decoration: none;
+
+    &:hover {
+      background-color: #253b60
+    }
+   }
+  }
+}
+
+.home-main {
+  margin-top: 60px;
+  background-image: url(/src/ui/Icons//png-icons/wave.svg), linear-gradient(60deg, rgb(82, 67, 170), rgb(237, 80, 180));
+  background-position: center calc(100% + 120px), 0 0;
+  background-repeat: no-repeat, no-repeat;
+  background-size: 100% auto, cover;
+  color: #fff;
+  height: 67%;
+
+    .bg {
+      padding-top: 10%;
+      margin-left: auto;
+      margin-right: auto;
+      padding-left: 1rem;
+      padding-right: 1rem;
```

Comment:
padding-inline: 1rem; if you copy from trello please check what is the value in px.